torntools/scripts/content/bazaar/ttBazaar.js

```diff
@@ -53,25 +53,26 @@ requireDatabase().then(() => {
 					let span = doc.new({ type: "span", class: "not-bold", text: `Could not find item. Please try using the Search function.` });
 
 					div.appendChild(span);
-					doc.find(".info-msg-cont .msg").appendChild(div);
+					requireElement(".msg.right-round").then(() => doc.find(".msg.right-round").appendChild(div), undefined);
```

Comment:
This element should already be present?

Reply:
It gave me errors when it was present (like once every 2 visits). And hence I added that.

Reply:
Wouldn't that be because the `bazaarLoaded` function triggers too fast with your selector?